src/main.js

```diff
@@ -22,13 +27,15 @@ var main = function() {
     console.log('Skipping tick ' + Game.time + ' due to lack of CPU.');
     return;
   }
+
   brain.prepareMemory();
   brain.handleNextroom();
   brain.handleSquadmanager();
   brain.handleIncomingTransactions();
-
   Memory.myRooms = _.map(_.filter(Game.rooms, (r) => r.execute()), r => r.name);
-
+  if (config.visualizer.enabled && config.visualizer.refresh) {
```

Comment:
I think you don't have do define any mocking objects, just make sure they are not called if not available.